fastlane/Fastfile

```diff
@@ -0,0 +1,16 @@
+default_platform(:ios)
+
+platform :ios do
+  desc "Build SwiftLeeds scheme"
+  lane :build do
```

Comment:
To help us out a little in future, we can actually pass the configuration to this as an argument. This would enable us when it comes to it, to pass "Release" etc.

So something like this would be the minimal way of doing it:

```
lane :build do |options|
...
configuration: options[:configuration]
```


An alternate method is to pass in semantic types like "Dev" "TestFlight" and then switch on those in the ruby to set relevant configs, but I don't think we'll need to go that far!


You'd have to change the configuration of the actions yml to pass the configuration either way 👍🏻 



```
fastlane build configuration:debug
```